resources/styles/molecules/accordion/BpAccordion.vue

```diff
@@ -1,25 +1,31 @@
 <template>
-    <div :class="[block, isOpen ? '-open' : '-closed' ]">
-        <header>
-            <div
-                :id="`${id}-control`"
-                :class="`${block}__${headerElement}`"
-                :aria-expanded="`${isOpen}`"
-                :aria-controls="id"
-                tabindex="0"
-                role="button"
-                @click="open"
-                @keyup.enter="open"
-            >
-                <slot name="heading" />
-                <slot name="icon">
-                    <svg :class="`${block}__icon`" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none">
-                        <polyline points="6 9 12 15 18 9" />
-                    </svg>
-                </slot>
-            </div>
-        </header>
-        <transition
+    <button :class="[block, isOpen ? '-open' : '-closed']">
```

Comment:
You shouldn't use a button element for this, it doesn't make sense and messes with keyboard usability.

Is there a reason you changed it from div>header to button>div? Might even make sense to look at [details/summary](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) elements nowadays.